src/classes/CRLP_RollupUI_SVC.cls

```diff
@@ -147,9 +148,10 @@ public with sharing class CRLP_RollupUI_SVC {
 
         model.labels = getAppLabels();
         model.isCRLPEnabled = Customizable_Rollup_Settings__c.getOrgDefaults().Customizable_Rollups_Enabled__c;
+        model.isAdmin = STG_Panel.runningUserIsAdmin();
 
         //return early with just labels and CRLP enabled flag if CRLP is disabled
-        if (!model.isCRLPEnabled) {
+        if (!(model.isCRLPEnabled)) {
```

Comment:
Out of curiosity, why the extra set of parenthesis?

Reply:
I was previously testing both isCRLPEnabled and isSystemAdminUser, so the second set was previously needed.  Just forgot to remove the extra set.